deploy/cert-manager-csi-driver.yaml

```diff
@@ -77,11 +77,8 @@ spec:
         - name: cert-manager-csi
           securityContext:
             privileged: true
-            capabilities:
-              add: ["SYS_ADMIN"]
-            allowPrivilegeEscalation: true
```

Comment:
Why has this changed? Were these two settings unneccessary in the first place?

Reply:
Yes, they may not be necessary depending on how the driver is being used. I removed since it shouldn't be the default